test/indicators/slider_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:macos_ui/macos_ui.dart';
+
+void main() {
+  testWidgets('debugFillProperties', (tester) async {
```

Comment:
Are there any other tests we can do besides the debug props? Maybe take a look at what tests Flutter's material slider has?